.bash_prompt

```diff
@@ -103,14 +103,14 @@ else
 	hostStyle="${yellow}";
 fi;
 
-# Set the terminal title to the current working directory.
-PS1="\033]0; ${PWD##*/}\007";
+# Set the terminal title to the current working folder.
```

Comment:
directory => ~/Projects/dotfiles
folder => dotfiles

The problem with #623 was that it would embed the result of ${PWD##*/} within PS1, so you would stuck with same title during navigation between folders.

Original usage was \w, changed to ${PWD##*/} in #623 (comment was not updated to reflect change) and fixed now with \W.

Reference:
For path ~/Projects/dotfiles
\w => ~/Projects/dotfiles
\W => dotfiles